policyengine_us/variables/gov/irs/income/taxable_income/deductions/qualified_business_income_deduction/qualified_business_income.py

```diff
@@ -13,6 +13,12 @@ class qualified_business_income(Variable):
 
     def formula(person, period, parameters):
         p = parameters(period).gov.irs.deductions.qbi
-        gross_qbi = add(person, period, p.income_definition)
+        gross_qbi = sum(
```

Comment:
is `sum()` vectorized? if not, you could do this instead:
```suggestion
        gross_qbi = 0
        for var in p.income_definition:
            gross_qbi += person(var, period) * person(var + "_would_be_qualified", period)
```